modules/travel_pay/app/models/travel_pay/travel_pay_cache.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
+
+module TravelPay
+  class TravelPayTokenStore < Common::RedisStore
+    # include Common::CacheAside
+    redis_store REDIS_CONFIG[:travel_pay_token_store][:namespace]
+    redis_ttl REDIS_CONFIG[:travel_pay_token_store][:each_ttl]
```

Comment:
I don't know if we have the option to set the TTL, but if the default is 30 mins (1800s as expected in the test below), the btsss token is expiring twice as fast as it needs to

Reply:
Looks like we can control this, we'd just need to add it here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/redis.yml

Reply:
Yeah, that's where I set it to 1800. I just changed it to 3600, since that matches the expiration of the tokens.